package-lock.json

```diff
@@ -2784,8 +2784,7 @@
 		"node_modules/@types/lodash": {
 			"version": "4.14.201",
 			"resolved": "https://registry.npmjs.org/@types/lodash/-/lodash-4.14.201.tgz",
-			"integrity": "sha512-y9euML0cim1JrykNxADLfaG0FgD1g/yTHwUs/Jg9ZIU7WKj2/4IW9Lbb1WZbvck78W/lfGXFfe+u2EGfIJXdLQ==",
-			"dev": true
+			"integrity": "sha512-y9euML0cim1JrykNxADLfaG0FgD1g/yTHwUs/Jg9ZIU7WKj2/4IW9Lbb1WZbvck78W/lfGXFfe+u2EGfIJXdLQ=="
```

Comment:
Could the ``"dev": true`` line be put back in. Even if it is an appropriate change (would need to check), I don't think it should be part of this PR.

Reply:
Hello, I tried to pull before pushing the change but it seems I have accidentally tried to merge my forked development branch with the LG5 branch (?) and I can't push the change as it keeps asking me to merge. Is there a place to message you further to resolve this? I'm a new developer/contributor and I don't want to mess anything up or move forward without understanding what I'm doing. Thanks

Reply:
Disregard my last message, the dev: true should be back now, sorry for the confusion!